src/component/NoteModel/NoteModel.js

```diff
@@ -18,13 +19,30 @@ const tailLayout = {
   },
 };
 
-export default function NoteModel({visible = false, onOk = () => {}, setVisible = () => {}}) {
+export default function NoteModel({visible = false, setRefreshNotes = () => {}, setVisible = () => {}}) {
+
+  const [name,setName] = useState('');
```

Comment:
no need for this, on onFinish you get all filled values send that directly to the POST  API as body

Reply:
REmove all state